reverse-geocoder-processor/src/main/java/com/esri/geoevent/processor/reversegeocoder/ReverseGeocoderProcessorDefinition.java

```diff
@@ -74,7 +78,7 @@ public String getDomain()
 	@Override
 	public String getVersion()
 	{
-		return "10.5.0";
+		return "10.6.0";
```

Comment:
let's keep upgrading the version to 10.6.0 separate from this PR so we can include this enhancement with 10.5.x